### DISCUSSION TITLE
Commands related to windows do nothing, linux works fine

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2844] [Q&A]
So I tried to compile a thing on windows so my friend can play it. I tried lime test windows, lime build windows and they all do nothing. Like, actually nothing. I type them and then after a few seconds it just gives me ability to type again. 
![image](https://github.com/HaxeFlixel/flixel/assets/60806448/56d3e947-0e9d-47d8-a915-4b78e6c3eef7)
I tried doing the lime setup windows command, too, but it just does the same thing. However change windows to linux in all of these commands and they run just fine (i AM on linux by the way)
Idk what am I doing wrong. Please help

**Comment by NikRetaNCAM** [comment]
Update: it does compile to neko folder but it doesnt want to launch. Windows straight up tells 2 of my friends that it cannot be opened

**Comment by Geokureli** [comment]
try asking in the [openfl forum](https://community.openfl.org/)
